examples/3d_columns.rs

```diff
@@ -28,11 +31,35 @@ pub fn main() {
         .run();
 }
 
+#[derive(
+    Component, Clone, Debug, Default, Deref, DerefMut, Reflect, PartialEq, Eq, ExtractComponent,
+)]
+#[reflect(Component, Default)]
+pub struct StandardMaterialHandle(pub Handle<StandardMaterial>);
+
+impl From<Handle<StandardMaterial>> for StandardMaterialHandle {
+    fn from(handle: Handle<StandardMaterial>) -> Self {
+        Self(handle)
+    }
+}
+
+impl From<StandardMaterialHandle> for AssetId<StandardMaterial> {
+    fn from(material: StandardMaterialHandle) -> Self {
+        material.id()
+    }
+}
+
+impl From<&StandardMaterialHandle> for AssetId<StandardMaterial> {
+    fn from(material: &StandardMaterialHandle) -> Self {
+        material.id()
+    }
+}
```

Comment:
I don't think this type, added to most 3d examples is needed. we can keep storing a `Handle<StandardMaterial>` and instantiate a `MeshMaterial3d`